package.json

```diff
@@ -45,7 +45,7 @@
     "pino-pretty": "^10.3.1",
     "prettier": "^3.2.4",
     "ts-node": "^10.9.2",
-    "typescript": "^5.3.3"
+    "typescript": "^5.4.3",
```

Comment:
trailing comma is not allowed in json files

Reply:
my bad, I fixed it